### ISSUE TITLE
Is it possible to make a release and tag it?

### ISSUE THREAD

**Issue by rpanai** [robjhyndman/anomalous#7]
Hi Rob,
first of all thank you for your great packages. Do you think it's possible to make a release and tag it? As suggested [here](https://github.com/conda/conda/issues/6674#issuecomment-355814072)? I'm trying to build a conda package for this.

**Comment by robjhyndman** [comment]
You would be better off using the tsfeatures package which also computes features but is better designed and easy to extend. Is there anything in the anomalous package that you need which is not in tsfeatures? See https://pkg.robjhyndman.com/tsfeatures/

**Comment by rpanai** [comment]
Apparently the following are not defined in tsfeatures
* LShift 
* VChange
* KLScore

**Comment by robjhyndman** [comment]
All available in tsfeatures: https://pkg.robjhyndman.com/tsfeatures/reference/max_level_shift.html

**Comment by rpanai** [comment]
Thanks I'll check it out.

**Comment by rpanai** [comment]
It looks to me that `anomalous` and `tsfeatures` don't return the same output

# `anomalous`
```R
library(anomalous)
df <- read.csv("https://gist.githubusercontent.com/rpanai/48298bc1f4a58638b6a9c03f7781c08f/raw/19600ea8825b8af90a1d54f74a8c46942ef26ab9/test.csv", header=FALSE)
tsmeasures(df)
```
```
     lumpiness   entropy      ACF1   lshift  vchange cpoints fspots      trend
[1,] 0.6504501 0.9245034 0.2700043 1.422641 1.219153      87      6 0.05775423
     linearity curvature    spikiness  KLscore change.idx
[1,] -3.448447 0.7236265 5.877808e-06 10.37368         79
attr(,"class")
[1] "features" "matrix" 
```

# `tsfeatures`
```R
library(tsfeatures)
df <- read.csv("https://gist.githubusercontent.com/rpanai/48298bc1f4a58638b6a9c03f7781c08f/raw/19600ea8825b8af90a1d54f74a8c46942ef26ab9/test.csv", header=FALSE)

as.matrix(tsfeatures(df))
```
```
     frequency nperiods seasonal_period      trend        spike linearity
[1,]         1        0               1 0.06408193 5.816305e-06 -3.373296
     curvature    e_acf1    e_acf10   entropy    x_acf1   x_acf10 diff1_acf1
[1,] 0.3717988 0.2203592 0.06274414 0.9841804 0.2700043 0.1276507 -0.3657909
     diff1_acf10 diff2_acf1 diff2_acf10
[1,]   0.1696449 -0.5855102   0.3639894
```

In particular on the first case we have curvature = 0.7236265 while on the second curvature = 0.3717988

**Comment by robjhyndman** [comment]
We've improved the computation of these features over time, and `tsfeatures` had some updated estimation, particularly for features involving a time series decomposition first. In `anomalous`, when the series was non-seasonal (as in this example), we used `fitted(mgcv::gam(y ~ s(t)))` to estimate the trend. In `tsfeatures`, the same trend was estimated using `stats::supsmu(t,y)`. In our latest package, `feasts`, we also use `stats::supsmu(t,y)`.

Some other features have also had updated estimation procedures. See the source code for any details.
